src/index.js

```diff
@@ -410,6 +424,11 @@ Dropzone.propTypes = {
   disableClick: PropTypes.bool,
 
   /**
+ * Enable/disable the dropzone entirely
+ */
+  disableDropzone: PropTypes.bool,
```

Comment:
Let's rename the prop to `disabled` to conform the DOM API: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button